packages/components/app/styles/components/dropdown.scss

```diff
@@ -408,8 +416,82 @@ $hds-dropdown-toggle-border-radius: 5px;
   }
 }
 
+// HDS::DROPDOWN::LIST-ITEM::TITLE
+
 .hds-dropdown-list-item--title {
   padding: 10px 16px 4px;
   color: var(--token-color-foreground-strong);
 }
 
+// HDS::DROPDOWN::LIST-ITEM::CHECKMARK
+
+.hds-dropdown-list-item--checkmark {
+  color: var(--token-color-foreground-action-active);
+
+  a.hds-dropdown-list-item__interactive,
+  button.hds-dropdown-list-item__interactive {
+    // leave room for the checkmark to avoid overlapping with text
+    padding-right: 30px;
+
+    // disable background on focus and active states
+    &:focus,
+    &:active,
+    &.mock-focus,
+    &.mock-active {
+      &::after {
+        background-color: inherit;
+      }
+    }
+  }
+
+  .hds-dropdown-list-item__interactive-text {
+    flex: 1;
+  }
+}
+
+.hds-dropdown-list-item--checkmark-selected {
+  background-image: url("data:image/svg+xml,%3Csvg viewBox='0 0 16 16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M14.78 4.28a.75.75 0 00-1.06-1.06l-7.97 7.97-3.47-3.47a.75.75 0 00-1.06 1.06l4 4a.75.75 0 001.06 0l8.5-8.5z' fill='%231060ff'/%3E%3C/svg%3E");
+  background-repeat: no-repeat;
+  background-position: top 10px right 10px;
+  background-size: var(--token-form-select-background-image-size) var(--token-form-select-background-image-size);
+
+  .hds-dropdown-list-item__interactive {
+    color: var(--token-color-foreground-action-active);
+  }
+}
+
+// HDS::DROPDOWN::LIST-ITEM::CHECKBOX & HDS::DROPDOWN::LIST-ITEM::RADIO
+
+.hds-dropdown-list-item--checkbox,
+.hds-dropdown-list-item--radio {
+  display: flex;
+  padding: 8px 16px;
+
+  .hds-form-field--layout-flag {
+    flex: 1;
+  }
+
+  .hds-form-label {
+    font-weight: normal;
```

Comment:
In the designs in Figma, I see the color is not `form/label` but `foreground/primary`. Can you check with @heatherlarsen which one we should use?

Reply:
`foreground/primary` is correct. This is to keep the color of the label when it is within the menu, consistent with the other dropdown list item variants, which use `foreground/primary`.

Reply:
In this case, another reason (not strong, just a possibility) to considering using the `Base` form element, and not the `Field`, like it's been done in the `Pagination` component (and in that case, it's likely you will not need to override the `hds-form-field--layout-flag` flex).

Reply:
good spot @didoo and thank you for confirming @heatherlarsen; will try with the `Base` element the to minimise the overrides

Reply:
updated to [use the form base element instead of form field](https://github.com/hashicorp/design-system/pull/1185/commits/4aefe474463b81fe1b281b8b9a85a3f7c9029281)